assets/js/components/dashboard/EventForm.tsx

```diff
@@ -142,19 +143,43 @@ const PresenterField = ({ inputList, inputSetter }: ListFieldProps) => {
   );
 };
 
-const ModeratorField = () => {
+const ModeratorField = ({ inputList, inputSetter }: ListFieldProps) => {
+  const [inputModerator, setInputModerator] = useState<string>("");
+
+  const handleAddButton = () => {
+    inputModerator.trim() && inputSetter([...inputList, inputModerator.trim()]);
+    setInputModerator("");
+  };
```

Comment:
I think we should verify here if the user passes a proper email. We could use a famous regex for it:
`/^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/`